README.md

```diff
@@ -23,20 +23,58 @@ To install or update this blueprint:
 npm install -g generator-jhipster-native
 ```
 
+To build a Native image, you need to install a JDK that is compatible with GraalVM. Please refer to the [GraalVM Release Notes](https://www.graalvm.org/release-notes/) and install the appropriate JDK. Using SDKMAN simplifies the installation process.
+
+```
+sdk install java 22.3.3.r17-grl
```

Comment:
```suggestion
sdk install java 20.0.2-graalce
```

The coordinates have changed recently to match the JDK version. Of course, we'll likely need to update this soon since JDK 21 will be released tomorrow.